src/Controller/ContactController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+class ContactController extends AbstractController
```

Comment:
add PHPDoc annotation for class definition and function definitions. Install Visual Studio code extension:

![image](https://user-images.githubusercontent.com/19302124/202657224-f21b9906-31ab-4717-9936-4300ae43f0dc.png)